src/libs/actions/User.js

```diff
@@ -57,8 +57,11 @@ function changePassword(oldPassword, password) {
         });
 }
 
+/**
+ * @return {Promise}
+ */
 function getBetas() {
-    API.User_GetBetas().then((response) => {
+    return API.User_GetBetas().then((response) => {
```

Comment:
Returning from an Action function (that is consumed OUTSIDE of the Action file) breaks our architecture.

Reply:
Oh fuck, you are right!

Reply:
It's a pain, but I see two alternatives:

1) Pass a flag to `getBetas(shouldCloseModal)` and then close the modals within the action
2) Add an Onyx key, subscribe in `LogInWithShortLivedTokenPage`